ramls/alert.raml

```diff
@@ -1,5 +1,5 @@
 #%RAML 1.0
-title: "Sample-Module"
+title: "Orders Storage"
```

Comment:
please update the "title" field in the documentation section for consistency with the other APIs

Reply:
Not Sure if its something with Github, this file is not reflecting the changes in the PR, if you click on "view File", it has the correct title